drivers/scheduler/scheduler.go

```diff
@@ -20,8 +20,9 @@ const (
 
 // Context holds the execution context of a test task.
 type Context struct {
-	UID string
-	App *spec.AppSpec
+	UID        string
+	App        *spec.AppSpec
+	KubeConfig string
```

Comment:
Context object represents an application running in the system like mysql, elasticsearch, etc.
Why would we pass a different kubeconfig for every app?

Reply:
We want to verify the same app is running on a remote cluster after migration